Documentation/coldstaking.md

```diff
@@ -3,31 +3,31 @@
 
 ## Motivation
 
-In order to produce blocks on Stratis network, a miner has to be online with running node and have its wallet open. This is necessary because at each time slot, the minor is supposed to check whether 1 of its UTXOs is eligible to be used as so-called coinstake kernel input and if so, it needs to use the private key associated with this UTXO in order to produce the coinstake transaction.
+In order to produce blocks on Stratis network, a miner has to be online with running node and have its wallet open. This is necessary because at each time slot, the miner is supposed to check whether 1 of its UTXOs is eligible to be used as so-called coinstake kernel input and if so, it needs to use the private key associated with this UTXO in order to produce the coinstake transaction.
 
 The chance of a UTXO to be eligible for producing a coinstake transaction grows linearly with the number of coins that this UTXO presents.
 
 This implies that the biggest miners on the network are required to keep the coins in a hot wallet. This is dangerous in case the machine where the hot wallet runs is compromised.
 
-We propose cold staking, which is mechanism that eliminates the need to keep the coins in the hot wallet. With cold staking implemented, the minor still needs to be online with running the node and open wallet, but the coins that are used for staking, can be safely stored in cold storage. Therefore the open hot wallet does not need to hold any significant amount of coins, or it can even be completely empty.
+We propose cold staking, which is a mechanism that eliminates the need to keep the coins in the hot wallet. With cold staking implemented, the miner still needs to be online with a running node and open wallet, but the coins that are used for staking can be safely stored in cold storage. Therefore the open hot wallet does not need to hold any significant amount of coins, or it can even be completely empty.
 
 
 ## User interface flow
 
-We want to implement cold staking in a way that allows even inexperienced users to participate in production of new blocks. Since all the logic is implemented on the node side and the wallet is only using interfaces provided by the node, it is sufficient to describe the flow from inexperienced users point of view. Advanced users can then easily use the underlying functionality of the node.
+We want to implement cold staking in a way that allows even inexperienced users to participate in production of new blocks. Since all the logic is implemented on the node side and the wallet is only using interfaces provided by the node, it is sufficient to describe the flow from an inexperienced users point of view. Advanced users can then easily use the underlying functionality of the node.
```

Comment:
```suggestion
We want to implement cold staking in a way that allows even inexperienced users to participate in production of new blocks. Since all the logic is implemented on the node side and the wallet is only using interfaces provided by the node, it is sufficient to describe the flow from inexperienced users point of view. Advanced users can then easily use the underlying functionality of the node.
```